app/src/main/java/ch/admin/bag/dp3t/util/NotificationUtil.java

```diff
@@ -7,23 +7,32 @@
  *
  * SPDX-License-Identifier: MPL-2.0
  */
-
 package ch.admin.bag.dp3t.util;
 
 import android.app.Notification;
 import android.app.NotificationChannel;
 import android.app.NotificationManager;
+import android.app.PendingIntent;
 import android.content.Context;
+import android.content.Intent;
 import android.os.Build;
 import androidx.annotation.RequiresApi;
+import androidx.core.app.NotificationCompat;
+import androidx.core.app.TaskStackBuilder;
 
+import ch.admin.bag.dp3t.MainActivity;
 import ch.admin.bag.dp3t.R;
 
 public class NotificationUtil {
 
+	public static final String ACTION_ACTIVATE_TRACING = "ACTION_ACTIVATE_TRACING";
+
 	public static final String NOTIFICATION_CHANNEL_ID = "contact-channel";
+	public static final String CHANNEL_ID_REMINDER = "CHANNEL_ID_REMINDER";
 	public static final int NOTIFICATION_ID_CONTACT = 42;
 	public static final int NOTIFICATION_ID_UPDATE = 43;
+	public static final int NOTIFICATION_ID_REMINDER = 44;
+
 
 	@RequiresApi(api = Build.VERSION_CODES.O)
 	public static void createNotificationChannel(Context context) {
```

Comment:
We should unify this method with the new method (call new method from this method with specific arguments).